composer.json

```diff
@@ -26,7 +26,8 @@
     "require-dev": {
         "symfony/yaml": "~3.4 || ~4.0 || ^5.0",
         "symfony/console": "~3.4 || ~4.0 || ^5.0",
-        "symfony/phpunit-bridge": "^3.4.19 || ^4.0 || ^5.0"
+        "symfony/phpunit-bridge": "^3.4.19 || ^4.0 || ^5.0",
+        "phpunit/phpunit": "<8.0"
```

Comment:
why require phpunit? it makes use of symfony/phpunit-bridge which installs phpunit itself in a version compatible with php